src/Artsy/SystemContext.tsx

```diff
@@ -19,6 +22,11 @@ export interface SystemContextProps {
    */
   mediator?: Mediator
 
+  /**
+   * FIXME: Ask alloy how to pass one-off props like this in from force
+   */
```

Comment:
(Ignore this `FIXME`; spoke over slack today and going to investigate something in another PR.)